pyproject.toml

```diff
@@ -24,6 +24,7 @@ packages = ["src/biomapper2"]
 dev = [
     "ipykernel>=7.1.0",
     "black>=25.11.0",
+    "ipykernel>=7.1.0",
```

Comment:
it looks like `ipykernel` is already in our toml (two lines above), so this is a duplicate - should be removed